docs/documentation.md

```diff
@@ -785,8 +798,8 @@ Reason why the JamSession was closed.
 
 | Reason      	    | Description       
 |----------	        |-------------------
-| ``host``          | The Host closed the JamSession
-| ``inactive`` 	    | The JamSession was closed due to inactivity 
+| ``host``          | The *host* closed the *JamSession*.
+| ``inactive`` 	    | The *JamSession+ was closed due to inactivity. 
```

Comment:
Please press SHIFT when trying to type "*".

Reply:
Thank you. Will try to press the SHIFT key harder.